tools/travis-linux-arm_cross_compile.sh

```diff
@@ -27,27 +30,30 @@ EOF
 
 _set_up_environment() {
     # open-zwave wants -cc and -c++ but no package seems to provid them.
-    sudo cp /usr/bin/arm-linux-gnueabihf-gcc /usr/bin/arm-linux-gnueabihf-cc
-    sudo cp /usr/bin/arm-linux-gnueabihf-g++ /usr/bin/arm-linux-gnueabihf-c++
+    sudo cp "/usr/bin/$BUILD_TARGET-gcc" "/usr/bin/$BUILD_TARGET-cc"
+    sudo cp "/usr/bin/$BUILD_TARGET-g++" "/usr/bin/$BUILD_TARGET-c++"
 
     # For rust-crypto
-    export CC=arm-linux-gnueabihf-gcc
+    export CC="$BUILD_TARGET-gcc"
 
     # For open-zwave
-    export CROSS_COMPILE=arm-linux-gnueabihf-
+    export CROSS_COMPILE="$BUILD_TARGET-"
 
     tee -a $HOME/.cargo/config << EOF
-[target.armv7-unknown-linux-gnueabihf]
-linker = "arm-linux-gnueabihf-gcc"
+[target.$RUST_TARGET]
+linker = "$BUILD_TARGET-gcc"
 EOF
+
+    export PKG_CONFIG_LIBDIR="/usr/lib/$BUILD_TARGET/pkgconfig"
+    export PKG_CONFIG_ALLOW_CROSS=1
```

Comment:
@fabricedesre This line and the one above will be needed in the Docker image https://hub.docker.com/r/fabricedesre/foxbox-armhf/